uber/sep_commands.py

```diff
@@ -4,13 +4,41 @@
 from os.path import join
 from pprint import pprint
 
-from sideboard.lib import entry_point
 from sqlalchemy.orm import subqueryload
 
-from uber.config import c
+from uber.config import c, plugins_dir
 from uber.decorators import timed
 from uber.models import AdminAccount, Attendee, AutomatedEmail, Group, Session
 
+def entry_point(func):
+    """
+    Decorator used to define entry points for command-line scripts.  Sideboard
```

Comment:
Another couple references to Sideboard here!

Reply:
...should we rename sep? uep?

Reply:
Oh man I had no idea the S was for Sideboard haha. Maybe we can backronym it? I like sep